application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -56,4 +60,27 @@ public static String escapeMarkdown(String text) {
         return afterEscape.replace("\\```", "\\`\\`\\`");
     }
 
+    /**
+     * Converts a slash command text to a mentioned slash command, which you can directly click on
+     * in Discord.
+     *
+     * @param guild the {@link Guild} that contains the command
+     * @param fullCommand the command's name with its optional subcommand
+     * @return Formatted string for the mentioned slash command
+     * @throws IllegalArgumentException when the command isn't found in the guild
+     */
+    public static RestAction<String> mentionSlashCommand(Guild guild, @NotNull String fullCommand) {
+        List<String> fullCommandSplit = List.of(fullCommand.split(" "));
```

Comment:
y r u making it a list?

Reply:
y even have all these just do
```java
String commandName = commandPath.split(" ")[0];
```
u never need the other 2 parts (if any)

Reply:
... in which case u should limit the `split`, i.e. `split(" ", 1)` 😛 @Taz03

Reply:
yea this too